docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -57,32 +57,41 @@ Refer to:
 * {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
 --
 
-[IMPORTANT] 
+[IMPORTANT]
 .Restrictions in {serverless-short}
-==== 
+====
 If you are using {agent} with link:{serverless-docs}[{serverless-full}], note these differences from use with {ess} and self-managed {es}:
 
 * The number of {agents} that may be connected to an {serverless-full} project is limited to 10 thousand.
 * The minimum supported version of {agent} supported for use with {serverless-full} is 8.11.0.
 ====
 
 [discrete]
-[[elastic-agent-installation-minimum-requirements]]
-== Minimum requirements  
+[[elastic-agent-installation-resource-requirements]]
+== Resource requirements
+
+The {agent} resources consumption is influenced by the number of integration and the environment its been running on.
+
+Using our lab environment as an example, we can observe the following resource consumption:
 
-The following are the minimum system requirements for installing {agent}.
 // lint ignore mem
 [discrete]
 === CPU and RSS memory size
 
-// lint ignore 2vcpu 1gb
-Minimum requirements have been determined by running the {agent} on a GCP `e2-micro` instance (2vCPU/1GB).
-The {agent} used the default policy, running the system integration and self-monitoring.
-
+// lint ignore 2 vCPU 16.0 GiB
+We tested using an AWS `m7i.xlarge` instance type with 4 vCPUs, 16.0 GB of memory, and up to 12.5 Gbps of bandwidth. The tests ingested a single log file using both the <<output-elasticsearch-performance-tuning-settings,throughput and scale preset>> with self monitoring enabled.
```

Comment:
remind to self update the instance type, this instance type is the initial testing